<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>771</width>
    <height>444</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::ActionsContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>ADFG(V)X šifra</string>
  </property>
  <widget class="QRadioButton" name="radioButtonSiforvat">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>380</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Šifrovat</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonOdsifrovat">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>400</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Odšifrovat</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelVstupniData">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Vstupní data</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelVystupniData">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Výstupní data</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonProved">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>390</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Vypočítej</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonNacist">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>40</y>
     <width>151</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Načíst text ze souboru</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonUloz">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>220</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Uložit výstup</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelklic">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Klíč:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditklic">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>391</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEditVstup">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>421</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEditVystup">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>421</width>
     <height>171</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxjazyk">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>340</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Česká abeceda</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Anglická abeceda</string>
    </property>
   </item>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>40</y>
     <width>221</width>
     <height>181</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustIgnored</enum>
   </property>
   <property name="rowCount">
    <number>5</number>
   </property>
   <property name="columnCount">
    <number>5</number>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>39</number>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>29</number>
   </attribute>
   <attribute name="verticalHeaderMinimumSectionSize">
    <number>23</number>
   </attribute>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>10</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Abeceda</string>
   </property>
  </widget>
  <widget class="QPushButton" name="zmenaTabulky">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>310</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Abeceda 6x6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonuloztabulku">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>270</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Uložit tabulku</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonnactitabulku">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>270</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Načíst tabulku</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonclear">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>270</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Vyčisti tabulku</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
